SimonGame/mukherjee16swastika/README.md

```diff
@@ -0,0 +1,11 @@
+# SimonSays
+It is a memory game which can be played between the player and computer . 
+The computer flashes a color which the player have to flash back in right sequence. 
+It is bulid using HTML, CSS and JS.
+It is a simple game , to boost our memory and concentration.
+
+# Screen Shot
+![image](https://github.com/thinkswell/javascript-mini-projects/assets/139066755/d5f72b76-d77f-4c94-ab44-b48588772c86)
+
+
```

Comment:
A live link would be better.
You can use github pages, for that if you want for live link.
Thank you.
I shall then merge it.

Reply:
Also make sure you follow community guidelines.